db/schema.rb

```diff
@@ -1500,50 +1500,6 @@
     t.index ["decidim_organization_id"], name: "index_verifications_csv_census_to_organization"
   end
 
-  create_table "equity_composites", primary_key: "gid", id: :serial, force: :cascade do |t|
```

Comment:
I am unsure why this happened. New to rails so it seems... odd that this disappeared

Reply:
This has also been happening to me whenever I run migrations - need to look into it, but this change shouldn't be committed

Reply:
(To clarify, the reason this is happening is because the database itself doesn't include this table, but I think it should so I'm not sure why it doesn't.)

Reply:
This file was modified in the same way for me when I ran the `script/dev-setup` script (outside of Docker)